components/SuggestedColor/index.js

```diff
@@ -0,0 +1,9 @@
+import { StyledDiv } from "./styles";
+
+export default function SuggestedColor({ text, color }) {
+  return (
+    <>
+      <StyledDiv color={color}>{text}</StyledDiv>
+    </>
```

Comment:
- Since this component is your only child element, you do not need the fragments.
- you could think about whether you really need a separate component (`SuggestedColor`).

```suggestion
      <StyledDiv color={color}>{text}</StyledDiv>
```